api/src/main/java/org/openmrs/api/OrderSetService.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api;
+
+import org.openmrs.OrderSet;
+import org.openmrs.annotation.Authorized;
+import org.openmrs.api.db.OrderSetDAO;
+import org.openmrs.util.PrivilegeConstants;
+
+import java.util.List;
+
+/**
+ * Contains methods pertaining to creating/deleting/voiding Order Sets.
+ * 
+ * @since 1.12
+ */
+public interface OrderSetService extends OpenmrsService {
+	
+	/**
+	 * Setter for the OrderSet data access object. The dao is used for saving and getting orders
+	 * to/from the database
+	 *
+	 * @param dao The data access object to use
+	 */
+	void setOrderSetDAO(OrderSetDAO dao);
+	
+	/**
+	 * Save or update the given <code>orderSet</code> in the database. If the OrderSet is retired
+	 * it will set retired by and retired date.
+	 * If OrderSetMembers are voided, it will set voided by and voided date.
```

Comment:
This line of javadoc seems wrong: OrderSetMember is OpenmrsMetadata so it can be retired, not voided.

That said, do we actually _want_ it to be retirable? Or should you just delete it with no auditing? (OrderSet itself should be retirable, but I'm not sure about each member.) @bmamlin ? (I don't feel strongly about this myself, so I think you should fix the javadoc line, but can ignore the rest of this comment.)